jobbergate-slurp/slurp/s3_ops.py

```diff
@@ -38,6 +43,61 @@ def build_managers():
         ),
     )
 
+    class NextGenApplicationFiles(ApplicationFiles):
```

Comment:
These two classes are exactly the same. To me, this is a code smell that leaves me wondering if inheritance is the right model to use here (instead of composition). Let's not change it now, but let's keep that in mind.

Reply:
Removed on the speed up refactoring https://github.com/omnivector-solutions/jobbergate/pull/181